src/lib/accessibility/FocusZone/FocusTrapZone.tsx

```diff
@@ -117,9 +117,16 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}>
     }
 
     this._unsubscribeFromEvents()
+    const lastActiveFocusTrap = FocusTrapZone._focusStack[FocusTrapZone._focusStack.length - 1]
 
     if (!FocusTrapZone._focusStack.length) {
       this._showContentInAccessibilityTree()
+    } else if (
+      lastActiveFocusTrap._root.current &&
+      lastActiveFocusTrap._root.current.hasAttribute(HIDDEN_FROM_ACC_TREE)
+    ) {
+      lastActiveFocusTrap._root.current.removeAttribute(HIDDEN_FROM_ACC_TREE)
+      lastActiveFocusTrap._root.current.removeAttribute('aria-hidden ')
```

Comment:
Not sure about the space in `'aria-hidden '`

Reply:
ha, nice catch. Thanks!